Makefile

```diff
@@ -0,0 +1,24 @@
+# https://gist.github.com/alexedwards/3b40775846535d0014ab1ff477e4a568
+
+## help: print this help message
+.PHONY: help
+help:
+	@echo 'Usage:'
+	@sed -n 's/^##//p' ${MAKEFILE_LIST} | column -t -s ':' |  sed -e 's/^/ /'
+
+## tidy: format code and tidy modfile
+.PHONY: tidy
+tidy:
+	go fmt ./...
+	go mod tidy -v
+
+## test: run all tests
+.PHONY: test
```

Comment:
Nice! We should add a command that runs golangci-lint to match the [Github Workflow](https://github.com/Netflix/spectator-go/blob/main/.github/workflows/release.yml#L37)